docs/api/covidcast.md

```diff
@@ -153,6 +153,15 @@ You should specify only one of these three parameters in any given query.
 remain the same as a previous issue. If you have a research problem that would
 require knowing when we last confirmed an unchanged value, please get in touch.
 
+#### Useful Syntax Features
+
+The following params help specify multiple source-signal, timetype-timevalue or geotype-geovalue pairs. Use them in replacement of the usual required params.
+
+| Parameter | Format | Description | Example |
+| --- | --- | --- | --- |
+| `signal` | `signal={source}:{signal1},{signal2}` | Specify multiple source-signal pairs, grouped by source | `signal=src1:sig1`, `signal=src1:sig1,sig2`, `signal=src1:*`, `signal=src1:sig1;src2:sig3` |
+| `time` | `time={timetype}:{timevalue1},{timevalue2}` | Specify multiple timetype-timevalue pairs, grouped by timetype | `time=day:*`, `time=day:20201201`, `time=day:20201201,20201202`, `time=day:20201201-20201204`, `time=day:*;day:20201201` |
```

Comment:
i would remove the last `time` example, its syntactically valid but semantically redundant
```suggestion
| `time` | `time={timetype}:{timevalue1},{timevalue2}` | Specify multiple timetype-timevalue pairs, grouped by timetype | `time=day:*`, `time=day:20201201`, `time=day:20201201,20201202`, `time=day:20201201-20201204` |
```